server/src/github.ts

```diff
@@ -108,6 +114,28 @@ export class GitHub {
       yield result.data;
     }
   }
+
+  // tslint:disable-next-line:no-any
+  async get(path: string, userToken: string, parseJSON = true): Promise<any> {
```

Comment:
Why would we ever want `parseJSON = false`?

Reply:
I use it in this PR, where reading the status code response is important to determining the type of response. The `parseJSON` flag is passed along to `request-promise-native`, which auto parses successful responses and throws a rejection on any non-200 status code.